packages/block-library/src/gallery/edit.js

```diff
@@ -572,6 +636,7 @@ export default function GalleryEdit( props ) {
 								columns: undefined,
 								imageCrop: true,
 								randomOrder: false,
+								aspectRatio: 'auto',
 							} );
 
```

Comment:
I believe we need this to get resetting to work correctily:
```suggestion

setAspectRatio( 'auto' );
```